test/Uds-Config-Tool/Unit Tests/minlengthonly.odx

```diff
@@ -0,0 +1,249 @@
+                      </COMPU-CONST>
+                    </COMPU-SCALE>
+                    <COMPU-SCALE>
+                      <LOWER-LIMIT>49</LOWER-LIMIT>
+                      <UPPER-LIMIT>49</UPPER-LIMIT>
+                      <COMPU-CONST>
+                        <VT>Request out of range</VT>
+                      </COMPU-CONST>
+                    </COMPU-SCALE>
+                    <COMPU-SCALE>
+                      <LOWER-LIMIT>51</LOWER-LIMIT>
+                      <UPPER-LIMIT>51</UPPER-LIMIT>
+                      <COMPU-CONST>
+                        <VT>Security access denied</VT>
+                      </COMPU-CONST>
+                    </COMPU-SCALE>
+                  </COMPU-SCALES>
+                </COMPU-INTERNAL-TO-PHYS>
+              </COMPU-METHOD>
+              <DIAG-CODED-TYPE BASE-DATA-TYPE="A_UINT32" xsi:type="STANDARD-LENGTH-TYPE">
+                <BIT-LENGTH>8</BIT-LENGTH>
+              </DIAG-CODED-TYPE>
+              <PHYSICAL-TYPE BASE-DATA-TYPE="A_UNICODE2STRING"/>
+              <INTERNAL-CONSTR>
+                <SCALE-CONSTRS>
+                  <SCALE-CONSTR VALIDITY="NOT-DEFINED">
+                    <LOWER-LIMIT>0</LOWER-LIMIT>
+                    <UPPER-LIMIT>18</UPPER-LIMIT>
+                  </SCALE-CONSTR>
+                  <SCALE-CONSTR VALIDITY="NOT-DEFINED">
+                    <LOWER-LIMIT>20</LOWER-LIMIT>
+                    <UPPER-LIMIT>33</UPPER-LIMIT>
+                  </SCALE-CONSTR>
+                  <SCALE-CONSTR VALIDITY="NOT-DEFINED">
+                    <LOWER-LIMIT>35</LOWER-LIMIT>
+                    <UPPER-LIMIT>48</UPPER-LIMIT>
+                  </SCALE-CONSTR>
+                  <SCALE-CONSTR VALIDITY="NOT-DEFINED">
+                    <LOWER-LIMIT>50</LOWER-LIMIT>
+                    <UPPER-LIMIT>50</UPPER-LIMIT>
+                  </SCALE-CONSTR>
+                  <SCALE-CONSTR VALIDITY="NOT-DEFINED">
+                    <LOWER-LIMIT>52</LOWER-LIMIT>
+                    <UPPER-LIMIT>255</UPPER-LIMIT>
+                  </SCALE-CONSTR>
+                </SCALE-CONSTRS>
+              </INTERNAL-CONSTR>
+            </DATA-OBJECT-PROP>
+          </DATA-OBJECT-PROPS>
+          <STRUCTURES>
+            <STRUCTURE ID="_2">
+              <SHORT-NAME>Dynamic_PartNumber</SHORT-NAME>
+              <LONG-NAME>Dynamic_PartNumber</LONG-NAME>
+              <PARAMS>
+                <PARAM SEMANTIC="DATA" xsi:type="VALUE">
+                  <SHORT-NAME>PartNumber</SHORT-NAME>
+                  <LONG-NAME>PartNumber</LONG-NAME>
+                  <BYTE-POSITION>0</BYTE-POSITION>
+                  <DOP-REF ID-REF="_1"/>
+                </PARAM>
+              </PARAMS>
+            </STRUCTURE>
+          </STRUCTURES>
+        </DIAG-DATA-DICTIONARY-SPEC>
+        <DIAG-COMMS>
+          <DIAG-SERVICE ID="_3" SEMANTIC="IDENTIFICATION">
+            <SHORT-NAME>DU_BoschDuPartNumber_Read</SHORT-NAME>
```

Comment:
Please remove any reference to private DIDs

Reply:
Don't know how I missed that. Fixed